doc/user-guide/components.rst

```diff
@@ -478,17 +478,17 @@ Custom Components
 
 If you want to define your own components and override the standard
 functionality of PyPSA, you can override the standard
-components by passing ``pypsa.Network()`` the arguments
-``override_components`` and ``override_component_attrs``.
-
-For this network, these will replace the standard definitions in
-``pypsa.components.components`` and
-``pypsa.components.component_attrs``, which correspond to the
-repository CSV files ``pypsa/components.csv`` and
-``pypsa/component_attrs/*.csv``.
-
-``components`` is a pandas.DataFrame with the component ``name``,
-``list_name`` and ``description``. ``component_attrs`` is a
-pypsa.descriptors.Dict of pandas.DataFrame with the attribute
-properties for each component.  Just follow the formatting for the
-standard components.
+components by passing the arguments ``override_components`` and 
+``override_component_attrs`` when initialising a network via 
+:meth:`pypsa.Network() <pypsa.Network>`.
```

Comment:
What happens now if I read a network from file which has custom components?

Reply:
Should work as before as `io.py` handles component data independently from network custom components in network initialization.